src/libsyntax/ext/expand.rs

```diff
@@ -551,11 +576,9 @@ impl<'a, 'b> MacroExpander<'a, 'b> {
 
         let opt_expanded = match *ext {
             DeclMacro(ref expand, def_span) => {
-                if let Err(msg) = validate_and_set_expn_info(def_span.map(|(_, s)| s),
```

Comment:
Could you add a test for macros 2.0 (`macro m { ... }`) as well?

Reply:
Wait, I see, the stability information is not filled for macros 2.0.

Reply:
It *can* be if we want it to.

Reply:
We certainly want it, stability of `macro` items not being checked is a bug in the same way as stability of `macro_rules` not being checked.